core/app/c/[communitySlug]/forms/[formSlug]/edit/page.tsx

```diff
@@ -28,14 +28,7 @@ export default async function Page(props: {
 		formSlug: string;
 		communitySlug: string;
 	}>;
-	searchParams: Promise<{
-		unsavedChanges: boolean;
-	}>;
 }) {
-	const searchParams = await props.searchParams;
-
-	const { unsavedChanges } = searchParams;
-
```

Comment:
this logic does not need to happen server side, we can just do it client side. this is much faster